website/docs/r/powerbidedicated_capacity.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "PowerBIDedicated"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_powerbidedicated_capacity"
+sidebar_current: "docs-azurerm-resource-powerbidedicated-capacity"
+description: |-
+  Manages a PowerBIDedicated Capacity.
```

Comment:
It seems that I cannot rename this description as what you suggested since this resource is to create capacity which is the feature of the power bi dedicated.